src/router/openai_proxy.py

```
@@ -0,0 +1,56 @@
+        headers["Authorization"] = authorization
+
+    async with httpx.AsyncClient(timeout=30.0) as client:
+        try:
+            resp = await client.post(UPSTREAM_URL, content=body, headers=headers)
+        except httpx.RequestError as e:
+            logger.exception("Upstream request failed: %s", e)
+            raise HTTPException(status_code=502, detail="upstream unavailable")
+
+    try:
+        return JSONResponse(status_code=resp.status_code, content=resp.json())
+    except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)